src/supertux/command_line_arguments.hpp

```diff
@@ -76,6 +76,8 @@ class CommandLineArguments
 
   boost::optional<std::string> repository_url;
 
+  boost::optional<std::string> editor_level;
```

Comment:
If we make `--edit-level` consistent with the current naming style, this could be `edit_level` (similar to `start_demo`, `play_demo`, and `start_level`).